app/models/webauthn_credential.rb

```diff
@@ -0,0 +1,8 @@
+class WebauthnCredential < ApplicationRecord
+  belongs_to :user
+
+  validates :external_id, uniqueness: true, presence: true
+  validates :public_key, presence: true
+  validates :nickname, presence: true
```

Comment:
Should nicknames be unique within each user? (eg. would devices with the same nickname be allowed?)